olive/engine/engine.py

```diff
@@ -309,7 +307,6 @@ def run(
                 1. Final model -> {output_dir}/{output_name}_{AcceleratorSpec}_model.onnx
                 2. JSON file -> {output_dir}/{output_name}_{AcceleratorSpec}_model.json
                 3. Evaluation results of the final model -> {output_dir}/{output_name}_{AcceleratorSpec}_metrics.json
-
```

Comment:
I add the blank lines to fix the warning when building docs

Reply:
I didn't realize this change. I'll add it back.